docs/pages/admin-guides/access-controls/access-requests/access-request-configuration.mdx

```diff
@@ -329,6 +329,15 @@ resource allowed by "root-node-access" they will be required to provide a reason
 role set includes multiple roles governing Access Requests to the same roles and resources,
 "require" mode takes precedence.
 
+## Custom request reason prompts
+
+As of right now it's only possible specify a custom prompt for all or none of the Access Requests
+for a particular user. A custom request prompt can be specified by assigned the user a role with
+`spec.options.request_prompt` set to non-empty string.
```

Comment:
I think it would make sense to include this information in the part of this guide where we talk about the `spec.options.request_prompt` field, rather than in a separate H2. The guide is organized according to categories of fields within the role configuration, and adding this H2 scrambles that organization a bit.